frontend/src/container/WorkspaceContent.jsx

```diff
@@ -263,6 +270,9 @@ class WorkspaceContent extends React.Component {
       })
     }
 
+    // INFO - GM - 2020/03/03 - add file breadcrumbs link if it exists
+    if (props.breadcrumbs.length === 4 && state.appOpenedType) breadcrumbsList.push(props.breadcrumbs[3])
```

Comment:
avoid usage of magic numbers.
If you have to use one, add comment explaining it.
"Magic numbers" are static int in code that don't explain themselves.
here, I see 4 and 3 (from breadcrumbs[3])